### PR TITLE
Added Language select filter to teacher group creation page.

### PR DESCRIPTION
**Pull request by stevemock**
Fixed bug with All button not deselecting properly in language selection widget.

### CODE REVIEW

File: webapp/src/main/webapp/src/app/shared/filter/student-filters.component.ts [hunk 1]
```diff
@@ -3,6 +3,8 @@ import { StudentFilterOptions } from './student-filter-options';
 import { StudentFilter } from './student-filter';
```
Comment (malaffoon) [inline webapp/src/main/webapp/src/app/shared/filter/student-filters.component.ts]: I'm confused: is this component not being used in both places? Just wondering why this change is happening now instead of earlier when this was added in the individual report screen.

Comment (stevemock) [reply webapp/src/main/webapp/src/app/shared/filter/student-filters.component.ts]: The sb-typeahead-group is being used in several places, yes. This particular instance of it just got missed in the original round of places it needed to go.

Comment (malaffoon) [reply webapp/src/main/webapp/src/app/shared/filter/student-filters.component.ts]: discussed offline, apparently adv-filters.component is similar to this component and i was confused
